src/test/resources/pages/instructorFeedbackRubricQuestionMoveColumnNewQuestionSuccess.html

```diff
@@ -883,19 +883,19 @@ <h5 class="col-sm-2">
                         </span>
                       </th>
                       <th class="rubricCol-1-0">
-                        <input class="form-control" disabled="" id="rubricChoice-1-0" name="rubricChoice-0" type="text" value="New Col 5 choice">
+                        <input class="form-control" disabled="" id="rubricChoice-1-0" name="rubricChoice-0" type="text" value="Col 5 Choice">
```

Comment:
If the changes were just refactoring of the test case, how come the html code is changing too?

Reply:
He made some changes to the test data.